source/RandomStock.h

```diff
@@ -0,0 +1,88 @@
+/* RandomStock.h
+Copyright (c) 2024 by David Stark
```

Comment:
You may not want to use your own name here, from what I've seen only Michael Zahniser did.
```suggestion
Copyright (c) 2024 by Zarkonnen
```